### ISSUE TITLE
Event update failed. Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt. 0 entries updated.  Sync finished!

### ISSUE THREAD

**Issue by ralf-wildermuth** [phw198/OutlookGoogleCalendarSync#1268]
> Checklist:-
> * [x] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [x] Reproduce the issue
> * [ ] Complete the information below
> * [x] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.9.0  
**Edition**: Portable  
**Sync Direction**: O->G

**Bug Description**  
When syncing to Google, fails with:
```
2021-08-26 07:58:52,004 ERROR  17 OutlookGoogleCalendarSync.Console [278] -  Event update failed.
2021-08-26 07:58:52,005 ERROR  17 OutlookGoogleCalendarSync.Console [278] -  Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
2021-08-26 07:58:52,021 ERROR  17 OutlookGoogleCalendarSync.OGCSexception [28] -  System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
2021-08-26 07:58:52,025 ERROR  17 OutlookGoogleCalendarSync.OGCSexception [21] -  Code: 0x80004003,-2147467261; Location: get_ActivePalette() in EventColour.cs:179:21
2021-08-26 07:58:52,026 ERROR  17 OutlookGoogleCalendarSync.OGCSexception [65] -     bei OutlookGoogleCalendarSync.GoogleOgcs.EventColour.get_ActivePalette() in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\GoogleOgcs\EventColour.cs:Zeile 179.
   bei OutlookGoogleCalendarSync.GoogleOgcs.EventColour.GetColour(String colourId) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\GoogleOgcs\EventColour.cs:Zeile 232.
   bei OutlookGoogleCalendarSync.GoogleOgcs.Calendar.UpdateCalendarEntry(AppointmentItem ai, Event ev, Int32& itemModified, Boolean forceCompare) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\GoogleOgcs\GoogleCalendar.cs:Zeile 741.
   bei OutlookGoogleCalendarSync.GoogleOgcs.Calendar.UpdateCalendarEntries(Dictionary`2 entriesToBeCompared, Int32& entriesUpdated) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\GoogleOgcs\GoogleCalendar.cs:Zeile 531.
```

**Any other information**  
:paperclip: Attach your `OGcalsync.log` file.
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/7052389/OGcalsync.log)

**Comment by phw198** [comment]
Seems the Google calendar you had set OGCS to sync with, you had then hidden from view in the webview.

I have therefore enhanced OGCS to retrieve hidden Google calendars and also provided an option to choose whether these are displayed in the settings tab drop down.

:package: This should now be resolved with [v2.9.0.2.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/7071551/v2.9.0.2.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

Please let me know how it goes.

**Comment by ralf-wildermuth** [comment]
So far v2.9.0.2 has been working fine - Thanks for the patch!
